src/main/java/sirius/biz/translations/BasicTranslations.java

```diff
@@ -142,14 +178,8 @@ public void updateText(@Nonnull Mapping field, String lang, String text) {
      * @param field {@link Mapping} of the translated field
      * @param lang  language code
      * @return {@link Optional} with translated text for the given language, or empty if none is found
-     * @throws IllegalArgumentException if lang is not supported by the system
      */
     public Optional<String> getText(@Nonnull Mapping field, String lang) {
-        if (!isSupportedLanguage(lang)) {
-            throw new IllegalArgumentException(
-                    "lang must be a language code supported by the system (supportedLanguages)!");
-        }
```

Comment:
Validity check for languages is already performed during the creation of a Translation entity.

Double-checking when getting texts currently leads to exceptions making affected pages unusable. Removed the exception, since returning an empty Optional is sufficient here and keeps pages functional.